lib/Core/Site/DomainObjectMapper.php

```diff
@@ -22,6 +23,8 @@
  */
 final class DomainObjectMapper
 {
+    use ContainerAwareTrait;
```

Comment:
Even if it's injected with a setter? Didn't try it myself, so that was just a random thought on top of my head.

As for using the trait, you can leave it, I was just wondering if there was a reason for using it.